FirebaseDynamicLinks/Tests/Unit/FIRDynamicLinksTest.m

```diff
@@ -1198,7 +1198,6 @@ - (void)testFailMatchesShortLinkFormatForCustomDomains {
 - (void)testPassMatchesShortLinkFormatForCustomDomains {
   NSArray<NSString *> *urlStrings = @[
     @"https://google.com/xyz", @"https://google.com/xyz/?link=https://somedomain",
-    @"https://google.com?link=https://somedomain", @"https://google.com/?link=https://somedomain",
```

Comment:
From the description I thought we were missing a type of link.  Why is a test being deleted versus adding an additional pattern?

Reply:
The link that we are testing here is a valid FDL link, but not a short link. I added this one when i did re writing the logic but its actually a long link. When I added back the missed regex validation, this test case is caught. So removing it now.